Final Project/Portfolio/contact.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Ashley Sullivan | Front End Developer &amp; Designer</title>
+	<meta name="description" content="">
+	<link rel="stylesheet" type="text/css" href="css/reset.css">
+	<link href='https://fonts.googleapis.com/css?family=Raleway:300,400,600' rel='stylesheet' type='text/css'>
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.5.0/css/font-awesome.min.css">
+	<script src="js/main.js"></script>
+</head>
+<body>
+	<header class="contact-header">
+		<div class="content">
+			<h1 class="contact-h1"><a href="index.html">Ashley Sullivan</a></h1>
```

Comment:
Some still do this, but "Contact" might be more of the <h1> on this page. Whatever describes THIS page the best.